client/my-sites/plans-v2/details.tsx

```diff
@@ -41,33 +44,27 @@ const DetailsPage = ( { duration, productSlug, rootUrl, header, footer }: Detail
 	const siteProducts = useSelector( ( state ) => getSiteProducts( state, siteId ) );
 	const products = useSelector( ( state ) => getProductsList( state ) );
 	const translate = useTranslate();
-	const root = rootUrl.replace( ':site', siteSlug );
-	const durationSuffix = duration ? '/' + durationToString( duration ) : '';
 	const isLoading = ! currencyCode || ( isFetchingProducts && ! products );
 
-	// If the product slug isn't one that has options, proceed to the upsell.
-	if ( ! ( PRODUCTS_WITH_OPTIONS as readonly string[] ).includes( productSlug ) ) {
```

Comment:
What's the reasoning behind removing this redirect?

Reply:
I'm so glad that you asked this because I removed those lines to test something and then I never put them back. Phew.